client/package.json

```diff
@@ -9,7 +9,9 @@
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "3.4.1",
+    "styled-components": "^5.1.1"
```

Comment:
This is my first time using styled-components, but will it limiting? I guess my main concern is just variables. Im currently using sass for the card components, and Ive added a lot of variables for consistent coloring/sizing.